queries/hoon/highlights.scm

```diff
@@ -0,0 +1,32 @@
+(number) @number
+
+(string) @string
+
+[
+  "("
+  ")"
+  "["
+  "]"
+]  @punctuation.bracket
+
+[
+  (coreTerminator)
+  (seriesTerminator)
+] @punctuation.delimiter
+
+
+(rune) @keyword
+
+(term) @constant
+
+(aura) @constant.builtin
+
+(Gap) @comment
+
+(boolean) @constant.builtin
+
+(date) @constant.builtin
+(mold) @constant.builtin
```

Comment:
```suggestion
(mold) @operator
```
Or `@punctuation.special`

Reply:
Here's the documentation for [molds](https://developers.urbit.org/reference/hoon/irregular#trivial-molds), it's a symbolic alias for a type, (noun, atom cell, boolean, null). So I think this should be `@symbol`